RecoLocalTracker/SiStripZeroSuppression/src/SiStripAPVRestorer.cc

```diff
@@ -130,18 +130,19 @@ uint16_t SiStripAPVRestorer::inspect(uint32_t detId, uint16_t firstAPV, const di
 
 void SiStripAPVRestorer::restore(uint16_t firstAPV, digivector_t& digis)
 {
-  if ( ForceNoRestore_ ) return;
+  if ( forceNoRestore_ ) return;
+  
+  const uint16_t nTotStripsPerAPV = 128;
```

Comment:
thanks for defining a constant.
However this definition is repeated in every method.
Please move it to the members and also use constexpr